src/utilities/bcl/templates/ReportingMeasure/tests/reporting_measure_test.rb

```diff
@@ -139,25 +140,28 @@ def test_with_drybulb_temp
     if File.exist?(report_path(test_name))
       FileUtils.rm(report_path(test_name))
     end
-    assert(!File.exist?(report_path(test_name)))
+    refute_path_exists(report_path(test_name))
 
     # temporarily change directory to the run directory and run the measure
-    start_dir = Dir.pwd
-    begin
-      Dir.chdir(run_dir(test_name))
-
-      # run the measure
+    Dir.chdir(run_dir(test_name)) do
       measure.run(runner, argument_map)
-      result = runner.result
-      show_output(result)
-      assert_equal('Success', result.value.valueName)
-      assert(result.warnings.empty?)
-    ensure
-      Dir.chdir(start_dir)
     end
 
+    result = runner.result
+    show_output(result)
+    assert_equal('Success', result.value.valueName)
+    assert_empty(result.stepWarnings)
+
+    sqlFile = runner.lastEnergyPlusSqlFile.get
+    if !sqlFile.connectionOpen
+      sqlFile.reopen
+    end
+    hours = sqlFile.hoursSimulated
+    refute_empty(hours)
+    assert_equal(8760.0, hours.get)
```

Comment:
That makes it clear that #5147 shouldn't happen if you do things correctly.

The reporting measure.rb does sql_file.close at the end.